.npmrc

```diff
@@ -0,0 +1,10 @@
+strict-peer-dependencies=false
+
+# packages/client
+# └─┬ yeoman-generator
+#   └─┬ github-username
+#     └─┬ @octokit/rest
+#       └─┬ @octokit/plugin-paginate-rest
+#         └── ✕ unmet peer @octokit/core@>=4: found 3.6.0 in @octokit/rest
+
+# hint: If you don't want pnpm to fail on peer dependency issues, add "strict-peer-dependencies=false" to an .npmrc file at the root of your project.
```

Comment:
Can we configure this with pnpm instead? 🤔 

Instead of putting the "whole" value to false

Reply:
Gone!